common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -267,7 +267,7 @@ def test_grader_from_conf(self):
             },
             {
                 'type': "Lab",
-                'min_count': 7,
```

Comment:
Rather than changing the existing test to deal with this edge case, I would love to see a separate test which only deals with this certain edge case.

Reply:
@awaisdar001 I wanted to avoid copying code. I thought it was better if this is integrated in an existing test. Just added this in case someone removes the conversion to int in the future the test fails. Also setting the min_count as a string is a mistake so I don't think we should have a test case for it.

Reply:
It's a good practice to have a test case for each scenario even though its an edge case or rare. 
The test case will lead to the fact that our system is capable of handling a certain edge case. So I would vote for redundancy is better than coupling here.